plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d

```diff
@@ -729,18 +769,9 @@ struct MetaSpan {
         immutable click_fmt = "onclick='ui_set_mut(%s)'";
         immutable click_fmt2 = "ui_set_mut(%s)";
         status = StatusColor.none;
-
+        // I can't see the use of these 'onclick' events.
         foreach (ref const m; muts) {
             status = pickColor(m, status);
-            if (onClick.length == 0 && m.mut.status == Mutation.Status.alive) {
```

Comment:
The onclick behavior removed.

Reply:
Two fold. You have found a technical debt, my fault. onClick should be removed and then onClick2 renamed to onClick.

The purpose of this is to prioritize what mutant is activated "first" when clicking on a mutation point. The user wants the "alive" mutant to be marked/highlighted because that is the "interesting" one to look at.

This is then used on line [206](https://github.com/joakim-brannstrom/dextool/blob/48143077494c005da659e56bb2929774b3c03233/plugin/mutate/source/dextool/plugin/mutate/backend/report/html/package.d#L206)